ospfd/ospf_vty.c

```diff
@@ -7023,6 +7029,21 @@ DEFUN (ip_ospf_area,
       return CMD_WARNING;
     }
 
+  // Check if we have an address arg and proccess it
+  if (argc == idx + 3) {
+    VTY_GET_IPV4_ADDRESS("interface address", addr, argv[(idx+2)]->arg);
```

Comment:
@qlyoung, oh yeah, I forgot about this. This should be a straight:

 `inet_aton(argv[idx+2]->arg, &addr)`

No need for additional checks, right?